webvr-migration.md

```diff
@@ -0,0 +1,298 @@
+
+When a WebVR application is done rendering it must call `vrDevice.submitFrame()` to capture the content rendered to the WebGL context’s default framebuffer and display it on the VR hardware.
+
+WebXR automatically presents the content of the [`XRWebGLLayer`](https://immersive-web.github.io/webxr/#xrwebgllayer-interface)’s [`framebuffer`](https://immersive-web.github.io/webxr/#dom-xrwebgllayer-framebuffer) to the VR hardware when the [`xrSession.requestAnimationFrame()`](https://immersive-web.github.io/webxr/#dom-xrsession-requestanimationframe) callback returns.
+
+**WebVR**
+```js
+let vrFrameData = new VRFrameData();
+function onFrame (t) {
+  // Queue a request for the next frame to keep the animation loop going.
+  vrDisplay.requestAnimationFrame(onFrame);
+
+  // Get the frame data, which contains the required matrices.
+  vrDisplay.getFrameData(frameData);
+
+  // Ensure we're rendering to the default backbuffer.
+  gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+
+  // Set the default left eye viewport, assuming that it wasn't changed during 
+  // the call to vrDisplay.requestPresent().
+  gl.viewport(0, 0, webglCanvas.width * 0.5, webglCanvas.height);
+
+  // Render the scene using a fictional rendering library with the left eye's
+  // projection and view matrix.
+  scene.setProjectionMatrix(frameData.leftProjectionMatrix);
+  scene.setViewMatrix(frameData.leftViewMatrix);
+  scene.render();
+
+  // Set the default right eye viewport.
+  gl.viewport(webglCanvas.width * 0.5, 0, webglCanvas.width * 0.5, webglCanvas.height);
+
+  // Render the scene using a fictional rendering library with the right eye's
+  // projection and view matrix.
+  scene.setProjectionMatrix(frameData.rightProjectionMatrix);
+  scene.setViewMatrix(frameData.rightViewMatrix);
+  scene.render();
+
+  vrDisplay.submitFrame();
+}
+```
+
+**WebXR**
+```js
+function onFrame(t, frame) {
+  // Queue a request for the next frame to keep the animation loop going.
+  xrSession.requestAnimationFrame(onXRFrame);
+
+  // Get the XRDevice pose relative to the Reference Space we created
+  // earlier.
+  let pose = frame.getViewerPose(xrReferenceSpace);
```

Comment:
Technically, this code should check if pose is null.  This is so totally why i filed #674 ;)